web/src/components/invite-modal/invite-modal.tsx

```diff
@@ -86,6 +92,46 @@ export default ({
         {copiedRecently ? 'Copied' : 'Copy link'}
         <FontIcon type="link" className="icon" />
       </Button>
+      <div className="or">or</div>
+      {/* TODO: If the client's default mailto handler is a web interface (eg.
+                Gmail), the current tab location will change before any
+                achievements are awarded. target="_blank" is not reliable for
+                mailto links.
+
+                Investigate adding a delay between triggering the submission
+                and actually submitting the form.  */}
```

Comment:
@phirework I'd like to hear your opinions on this, both from an experience standpoint and a technical standpoint. For good reason, web standards make it hard to delay a form submission, and even harder to trigger a mailto action. I'm sure we could find some workaround, but should we?